website/api/zkvm/developer-guide/profiling.md

```diff
@@ -75,7 +75,7 @@ Use the pprof web interface to compare the performance of the 3 Fibonacci implem
 Refer to the [pprof docs] for more inforamtion about the web interface.
 :::
 
-[profiling example]: https://github.com/risc0/risc0/examples/profiling
+[profiling example]: https://github.com/risc0/risc0/tree/main/examples/profiling
```

Comment:
This fixes next, which should point to main. But I think we need a fix for 0.19 as well